File: SEBASTIAN_CHAN_00550218/C++/Vaca.cpp
#include "Vaca.hpp"
#include <string>
#include <iostream>
using namespace std;
// Constructor
Vaca::Vaca(string n, double p, double l) : Mamifero(n, p), litrosDeLeche(l) {}

// Método para imprimir detalles de la vaca
void Vaca::imprimirV(){
    cout << "Nombre de la vaca: " << getNombre() << endl;
    cout << "Peso: " << getPeso() << endl;
    cout << "Litros de leche: " << litrosDeLeche << endl;
}

File: SEBASTIAN_CHAN_00550218/Pyth/Vaca.py
from Mamifero import Mamifero
import tkinter as tk
from tkinter import messagebox

class Vaca(Mamifero):
    def __init__(self, n, p, l):
        super().__init__(n, p)
        self.litrosDeLeche = l
    
    def imprimirV(self):
        messagebox.showinfo("Nombre de la vaca:", self.getNombre())
        messagebox.showinfo("Peso:", self.getPeso())
        messagebox.showinfo("Litros de leche:", self.litrosDeLeche)